src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -305,8 +334,9 @@ type FeeExplainerProps = CardProps
 
 export const FeeExplainer: React.FC<FeeExplainerProps> = props => {
   const votingPower = useAppSelector(selectVotingPower)
+  const sellAmountUsd = useAppSelector(selectSellAmountUsd)
 
-  const [tradeSize, setTradeSize] = useState(FEE_CURVE_NO_FEE_THRESHOLD_USD) // default to max below free so we have a value
+  const [tradeSize, setTradeSize] = useState(Number.parseFloat(sellAmountUsd ?? '0'))
```

Comment:
We probably want to keep this line as-is, and add an effect on `sellAmountUsd` that calls `setTradeSize` if gt(0)

Reply:
actioned in 19be7d23c47677edafae86e2457a7bc278d0e430v